test/test-generic-post.js

```diff
@@ -90,14 +90,7 @@ describe("check build output for a generic post", () => {
       expect(count).to.equal(1);
     });
 
-    it("should have a good CSP", () => {
-      const csp = select(
-        "meta[http-equiv='Content-Security-Policy']",
-        "content"
-      );
-      expect(csp).to.contain(";object-src 'none';");
-      expect(csp).to.match(/^default-src 'self';/);
-    });
+    // TODO: Check for a good CPS (_headers file)
```

Comment:
A quick sanity test would be awesome.

Reply:
Done. I added a test to check that the _header file exists, there is a CSP for the page and contains a good policy (as in the previous test for the CSP meta tag)